app/src/main/res/values-it/strings.xml

```diff
@@ -119,7 +127,10 @@
     <string name="dialogs_top_month">Top del mese</string>
     <string name="dialogs_top_week">Top della settimana</string>
     <string name="dialogs_top_year">Top dell\'anno</string>
+    <string name="dialogs_old">Vecchi</string>
     <string name="dialogs_unread">Non letti</string>
+    <string name="dialogs_server_version_outdated">La versione del server (%1$s) è inferiore alla minima supportata (%2$s).\\n\\nInforma i tuoi amministratori ed effettua il login su un\'altra istanza, o usa quella di default anonimamente.</string>
+    <string name="dialogs_server_version_outdated_short">La versione del server (%1$s) è troppo bassa.</string>
```

Comment:
I wonder if `vecchia` would be a better fit than `bassa`, given the context?

Reply:
Done